applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_adjoint_solver.py

```diff
@@ -64,8 +64,9 @@ def Initialize(self):
 
     def PrepareModelPart(self):
         super(PotentialFlowAdjointSolver, self).PrepareModelPart()
-       # defines how the primal elements should be replaced with their adjoint counterparts
-        replacement_settings = KratosMultiphysics.Parameters("""
+        # defines how the primal elements should be replaced with their adjoint counterparts
```

Comment:
Please, have a look to the mechanism in the FluidDynamicsApplication as well as the new potential_flow_solver. Try to fit into the current standards for the sake of code maintenance.

Reply:
I believe in this case @marcnunezc was following the standards of the StructuralMechanicsApplication, in particular the ones introduced in this file: applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_adjoint_static_solver.py. @marcnunezc I leave the decision to you whether you want to stick to this standard of the other one.

Reply:
@marcnunezc I think that you already took care of this in another PR. Am I wrong?

Reply:
Yes, in #5031!  I'd rather do this in the separate PR (if everyone agrees to do the change) in order to discuss the advantages/disadvantages of using one approach or the other.